lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -98,13 +99,10 @@ class FullPageScreenshot extends FRGatherer {
     await networkMonitor.enable();
 
     await session.sendCommand('Emulation.setDeviceMetricsOverride', {
-      // If we're gathering with mobile screenEmulation on (overlay scrollbars, etc), continue to use that for this screenshot.
-      mobile: context.settings.screenEmulation.mobile,
+      mobile: deviceMetrics.mobile,
       height,
-      width,
-      deviceScaleFactor: 1,
```

Comment:
the https://output.jsbin.com/koqamuh/4/quiet  test case does find a related problem in this branch..    

on master  the colorcontrast results work all the way up to 16300 px
![image](https://user-images.githubusercontent.com/39191/154616607-5175f279-ff49-4175-9039-8fbc785b244e.png)
but in this branch it wraps around at 16300/2: 
![image](https://user-images.githubusercontent.com/39191/154616689-d05d5038-5037-49c9-97e9-1837f4d970ff.png)
this wrapping around at maxtexturesize/dpr is behavior we worked around  in https://github.com/GoogleChrome/lighthouse/pull/11688 (i think)

Reply:
Yeah setting a DPR of 1 will also reduce the image "filesize". Going off https://github.com/GoogleChrome/lighthouse/issues/11121#issuecomment-705699860 I'm fine keeping the DPR at 1 for the FPS. The content change in https://github.com/GoogleChrome/lighthouse/issues/13642 appears to be from us changing the viewport width anyway.